CRM/CivirulesCronTrigger/RelationshipEnd.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+class CRM_CivirulesCronTrigger_RelationshipEnd extends CRM_Civirules_Trigger_Cron {
```

Comment:
I think it would have been better if this trigger is more flexible so it can be more useful for other scenarios. An example is the `CRM_CivirulesCronTrigger_MembershipEndDate`. I think this trigger should be named similarly i.e `CRM_CivirulesCronTrigger_RelationshipEndDate`. The flexibility of choosing if the trigger should be called some hours before/after/on the relationship end date will make this trigger to be more useful given the flexibility of other cron triggers in the extension.